src/main/java/org/axonframework/intellij/ide/plugin/annotator/AxonGutterAnnotator.java

```diff
@@ -33,53 +35,100 @@
 
     @Override
     public void annotate(@NotNull final PsiElement element, @NotNull AnnotationHolder holder) {
+        final PublisherProviderManager publisherManager = PublisherProviderManager.getInstance(element.getProject());
+        final HandlerProviderManager handlerManager = HandlerProviderManager.getInstance(element.getProject());
+        if (element instanceof PsiClass) {
```

Comment:
I think it would be nice to do something like:

if (element instanceof PsiClass)) {
  annotateClass()
} else if (element instanceof PsiMethodCallExpression || element instanceof PsiIdentifier) {
  annotateMethods()
}